src/openseadragon.js

```diff
@@ -2886,6 +2886,30 @@ function OpenSeadragon( options ){
         }
     }
 
+    /**
+     * Promise proxy in OpenSeadragon, can be removed once IE11 support is dropped
```

Comment:
IE11 support is already dropped as of #2300!

Reply:
Yeah, that's an old comment from a piece of code I had implemented locally, and forgot to remove. Since we don't have polyfils I thought this might be a good error-cacher, if someone has old version of some JS framework/browser. I can remove it...

Reply:
Hmm I've found out why I left it there:
>    332:58  error  Promise.resolve() is not supported in op_mini all  compat/compat

Reply:
So it's just Opera Mini? We don't explicitly support that browser, but I suppose if it's easy we might as well. It would be good to add an explicit comment that that's why this is here, then.